src/textual/_animator.py

```diff
@@ -27,14 +28,128 @@ def blend(self: T, destination: T, factor: float) -> T:
         ...
 
 
+def _in_out_expo(x):
```

Comment:
Could you type these? Seems redundant, but at some point I will switch on strict mode in Mypy and typing will be required.

Reply:
Of course. I would've guess I would be smart enough to look around, see everything is typed, and type those by myself... Turns out I'm not 🙃.